R/ssb.R

```diff
@@ -0,0 +1,112 @@
+#' Get SSB table meta data
+#'
+#' Fetch meta data of ready-made tables from Statistics Norway (SSB).
+#'
+#' @details
+#' The function returns a data.frame with an overview (meta data) of all the available ready-made
+#' tables produced by Statistics Norway.
+#'
+#' Note that the meta data is updated daily at 05.00 and 11.30 AM, which can lead to all tables
+#' being unavailable for up to five minutes.
+#'
+#' See the [API documentation](https://data.ssb.no/api/) for further details (in Norwegian only).
+#'
+#' @seealso [get_ssb_metadata_tags()] to view all meta data tags, and [get_ssb_table()]
+#'   to retrieve the actual tables.
+#' @return data.frame
+#' @rdname get_ssb_metadata
+#' @export
+#' @examples
+#' # Get data frame of all meta data
+#' get_ssb_metadata()
+#'
+#' # Get a subset of meta data which contains certain tags
+#' get_ssb_metadata(data_tag = "arbeidskraft")
+#'
+get_ssb_metadata <- function(url = "https://data.ssb.no/api/v0/dataset/list.csv?lang=no", data_tag = NULL) {
```

Comment:
I would rather we code the url as and internal variable. See data-raw/internal.R for how this is done for other data providers.